### PR TITLE
[panw] Map name for more subtypes, fix CSV escaping

### PR DESCRIPTION
**Pull request by chrisberkhout**
## Proposed commit message

```
[panw] Fix CSV escaping, map name for more subtypes (#)

The HTTP Headers field (`panw.panos.http_headers`) of the incoming data
is incorrectly escaped. This will be fixed if necessary before CSV
parsing.

Map the file name value in the URL/Filename (`panw.panos.misc`) field
for the `wildfire` and `wildfire-virus` sub-types.
```

## Discussion

The [documentation]() for the URL/Filename field says:

image

I've added the `wildfire` and `wildfire-virus` cases to the existing logic.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [ ] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices)

### CODE REVIEW

**Review by mjwolf — APPROVED** [review]
Looks good!

File: packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/threat.yml [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
 # API Docs: https://docs.paloaltonetworks.com/pan-os/10-2/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/threat-log-fields
 processors:
+  - script:
+      lang: painless
+      if: ctx.panw?.panos?.sub_type == 'url'
+      source: |
+        def fixHttpHeadersEscaping(String input) {
+          // Find a CSV fragment like `,Some-Header:"eg1.com, eg2.com";`
+          //     and correct it to be `,"Some-Header:""eg1.com, eg2.com"";"`
+          Matcher matcher = /,(([A-Za-z0-9\\-]+: *\"[^\"]*\"; *)+)/.matcher(input);
```
Comment (qcorporation) [inline packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/threat.yml:12]: @chrisberkhout - kind of controversial - do we not include support for underscore in header names?

Comment (chrisberkhout) [reply packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/threat.yml]: Good catch! I added that, and removed the backslash that was included accidentally.

**Review by qcorporation — APPROVED** [review]
Thank you for the change, this looks good

**Pull request merged**